### PR TITLE
HOCS-3972: fix extension and appeals extracts, add endpoint to get ac…

### PR DESCRIPTION
**Pull request by gary-newbolt**
…tions

### CODE REVIEW

File: src/main/java/uk/gov/digital/ho/hocs/info/api/TopicResource.java [hunk 1]
```diff
@@ -43,6 +43,7 @@ public TopicResource(TopicService topicService, ParentTopicRepository parentTopi
         return ResponseEntity.ok(GetAllTopicsResponse.fromTopicWithFilteredChildren(parentTopics, filteredChildTopics));
     }
 
+    // todo: remove
```
Comment (agraham19) [inline src/main/java/uk/gov/digital/ho/hocs/info/api/TopicResource.java:46]: Does this need removed?

File: src/test/resources/beforeTest.sql [hunk 2]
```diff
@@ -185,4 +186,4 @@ INSERT INTO entity (entity_list_UUID, uuid, simple_name, data, sort_order)
 VALUES  ('9fda236f-6cd4-4016-b4af-307c424eaa50','5abc65d9-3964-4c25-b570-46e203d5474b',
          'TEST_ENTITY_1', '{ "title" : "One"}'::jsonb, 1),
         ('9fda236f-6cd4-4016-b4af-307c424eaa50','8761bda1-8e26-4189-9133-aab3651aa584',
-         'TEST_ENTITY_2', '{ "title" : "Two"}'::jsonb, 2);
+         'TEST_ENTITY_2', '{ "title" : "Two"}'::jsonb, 2);
```
Comment (agraham19) [inline src/test/resources/beforeTest.sql:189]: This needs a newline

**Review by agraham19 — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/digital/ho/hocs/info/api/TopicResource.java (hunk 1 above)
Comment (gary-newbolt) [reply src/main/java/uk/gov/digital/ho/hocs/info/api/TopicResource.java:46]: Yeah... I put it there to remind me that the method needs to be removed however I didn't want to do it. I can though

File: src/test/resources/beforeTest.sql (hunk 2 above)
Comment (gary-newbolt) [reply src/test/resources/beforeTest.sql:189]: yes thanks... 
I'll add it on now.

**Review by agraham19 — APPROVED** [review]

**Pull request merged**